packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/native-classes/output.js

```diff
@@ -1,18 +1,37 @@
-var _foo, _bar;
+var Foo =
+/*#__PURE__*/
+function () {
+  "use strict";
```

Comment:
This is my mistake. This test case was meant to output `class` expressions, not compile classes to functions.

Reply:
That makes more sense indeed, I'll redo the whole test dir with that then!

Reply:
@jridgewell what should I do to convert tests to output class expressions?